noxfile.py

```diff
@@ -1,3 +1,9 @@
+"""Automatically build our documentation or run tests.
+
+Environments are re-used by default. Use the following-pattern to re-install them.
+
+nox -s docs -- -r
```

Comment:
❤️ I did not know this shorcut, I've been using `--no-reuse-existing-virtualenvs` which is so long to type that it hurts.